content/course-specific-material/introR-NDWSC-Nov17.Rmd

```diff
@@ -0,0 +1,67 @@
+---
+title: "ND WSC Intro to R Training"
+date: "2017-11-21"
+slug: "introR-NDWSC-Nov17"
+output: USGSmarkdowntemplates::hugoTraining_course
+parent: Course Specific Material
+---
+December 12-14, 2017 in Bismarck, ND
+
+### Installation
+
+See [R Installation Instructions](/installr) for information on what software should be installed prior to the course. 
+
+### Tentative schedule
+
+**Day 1**
+
+* 08:00 am - 08:30 am -- Instructors available for questions
+* 08:30 am - 09:30 am -- [Introduction](/intro-curriculum/Introduction)
+* 09:30 am - 10:00 am -- [Example workflow](/intro-curriculum/data/data_workflow.Rmd)
+* 10:00 am - 10:15 am -- *Break*
+* 10:15 am - 12:00 pm -- [Get](/intro-curriculum/Get)
+* 12:00 am - 01:00 pm -- *Lunch*
+* 01:00 pm - 03:00 pm -- [Clean](/intro-curriculum/Clean)
+* 03:00 pm - 03:15 pm -- *Break*
+* 03:15 pm - 04:00 pm -- More [Clean](/intro-curriculum/Clean)
+* 04:00 pm - 04:30 pm -- Project time!
+
+**Day 2**
+
+* 08:00 am - 08:30 am -- Instructors available for questions
+* 08:30 am - 10:15 am -- [Explore](/intro-curriculum/Explore)
+* 10:15 am - 10:30 am -- *Break*
+* 10:30 am - 12:00 pm -- [Analyze: Base](/intro-curriculum/Analyze)
+* 12:00 pm - 01:00 pm -- *Lunch*
+* 01:00 pm - 02:30 pm -- [Analyze: dataRetrieval](https://cran.r-project.org/web/packages/dataRetrieval/dataRetrieval.pdf) 
+* 02:30 pm - 02:45 pm -- *Break*
+* 02:45 pm - 04:00 pm -- Visualize with base R: [Visualize](/intro-curriculum/Visualize/)
+* 04:00 pm - 04:30 pm -- Project time!
+
+**Day 3**
+
+* 08:00 am - 08:30 am -- Instructors available for questions
+* 08:30 am - 10:00 am -- [Repeat](/intro-curriculum/Reproduce/)
+* 10:00 am - 12:00 pm -- Project time: [USGS R packages](/intro-curriculum/USGS/), projects (group/individual), or [additional topics](/intro-curriculum/Additional/)
```

Comment:
The third day is normally 8:30-12:30, but I had to change ours because of flights we needed to catch. Up to you if you want to change it back!